multiversx_sdk_cli/accounts.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, account_index: int = 0, address_index: int = 0) -> None:
     def sign_transaction(self, transaction: ITransaction) -> str:
         ledger_version = do_get_ledger_version()
         should_use_hash_signing = compare_versions(ledger_version, SIGN_USING_HASH_VERSION) >= 0
+
```

Comment:
Can we do something (in the next PR) about the mypy / reviewdog warnings shown in PRs?

Reply:
will take a look